lua/autorun/client/cl_nadmodpp.lua

```diff
@@ -23,6 +23,42 @@ net.Receive("nadmod_propowners",function(len)
 	end
 end)
 
+function NADMOD.GetPropOwner(ent)
+	local id = Props[ent:EntIndex()]
+	return id and player.GetBySteamID(id)
+end
+
+function NADMOD.PlayerCanTouch(ply, ent)
+	-- If PP is off or the ent is worldspawn, let them touch it
+	if not tobool(NADMOD.PPConfig["toggle"]) then return true end
+	if ent:IsWorld() then return ent:GetClass()=="worldspawn" end
+	if !IsValid(ent) or !IsValid(ply) or ent:IsPlayer() or !ply:IsPlayer() then return false end
+
+	local index = ent:EntIndex()
+	if not Props[index] then
+		return false
+	end
+
+	-- Ownerless props can be touched by all
+	if Props[index] == "O" then return true end 
+	-- Admins can touch anyones props + world
+	if NADMOD.PPConfig["adminall"] and NADMOD.IsPPAdmin(ply) then return true end
+	-- Players can touch their own props and friends
+	if Props[index] == ply:SteamID() then return true end
```

Comment:
This isn't checking friends.

Unsure how important false positives `true`'s might be in the Clientside PlayerCanTouch, but the comment at least isn't right anymore.

Theoretically, the client should know about its own friends at least... if we care.